include/openenclave/bits/result.h

```diff
@@ -265,10 +265,20 @@ typedef enum _oe_result
     OE_UNSUPPORTED_ENCLAVE_IMAGE,
 
     /**
-     * The CRL for a PCK certification expired
+     * The CRL for a PCK certificate has expired.
      */
     OE_VERIFY_CRL_EXPIRED,
 
+    /**
+     * The CRL for a PCK certificate could not be found.
```

Comment:
Ditto, "PCK" should be removed from line 273 unless it's added to 275

Reply:
Removed